workbench-core/authorization/src/caslAuthorizationPlugin.ts

```diff
@@ -23,12 +25,36 @@ export default class CASLAuthorizationPlugin implements AuthorizationPlugin {
       throw new ForbiddenError(err.message);
     }
   }
+
+  public async isAuthorizedOnDynamicOperations(
+    identityPermissions: IdentityPermission[],
+    dynamicOperations: DynamicOperation[]
+  ): Promise<void> {
+    const ability: Ability = this._defineAbilitiesForIdentityPermissions(identityPermissions);
+    try {
+      dynamicOperations.forEach((dynamicOperation: DynamicOperation) => {
+        const attributes = {
+          subjectId: dynamicOperation.subjectId,
+          ...dynamicOperation.subjectAttributes
+        };
+        const requestedSubject = subject(dynamicOperation.subjectType, attributes);
+        CASLForbiddenError.from(ability).throwUnlessCan(
```

Comment:
Does CASL allow you to test the positive case i.e. `throwIfCannot` instead of needing do an `unless` statement?